btr-web/btr-common-components/enums/session-storage-e.ts

```diff
@@ -0,0 +1,7 @@
+export enum SessionStorageKeyE {
```

Comment:
yeah its to keep these consistent with the other apps in case they end up using our components

Reply:
I agree and disagree, I would say it would be better to let them adapt then to have unstandardized stuff in one place. Because whenever someone is creating something new, will not know what/how to use it (happened to me more then once, can't figure out what is standard and what not)